source/includes/fact-mmapv1-big-endian.rst

```diff
@@ -1,5 +1,7 @@
 .. important::
 
-   MMAPv1 is not supported on big-endian architectures such as s390x.
-   MongoDB returns an error if you set MMAPv1 as the
-   storage engine on a big-endian system.
+   MMAPv1 is unsupported on big-endian architectures like the IBM
+   z/Architecture (``s390x``) and certain bi-endian architectures like
+   the PowerPC (``ppc64le``) platform. MongoDB returns an error if you
```

Comment:
We should keep this second section generic as well. Otherwise it implies MongoDB does *not* return errors for any other big-endian architecture, even if MMAPv1 isn't supported.

"MongoDB returns an error when starting a {{mongod}} using MMAPv1 on an unsupported architecture."